packages/terra-form/src/Fieldset.jsx

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable react/jsx-boolean-value, jsx-a11y/label-has-for */
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import 'terra-base/lib/baseStyles';
+
+import './Field.scss';
+
+const propTypes = {
+  /**
+   * Children the Field contains
+   */
+  children: PropTypes.node,
+  /**
+   * Error message for when the input is invalid
+   */
+  error: PropTypes.node,
+  /**
+   * Help element to display with the input
+   */
+  help: PropTypes.node,
+  /**
+   * The value for the htmlFor property on the label
+   */
+  htmlFor: PropTypes.string,
+  /**
+   * Determines whether the fieldset is an inline fieldset
+   */
+  isInline: PropTypes.bool,
+  /**
+   * Legend for the input group
+   */
+  legend: PropTypes.string,
+  /**
+   * Determines whether the fieldset is required
+   */
+  required: PropTypes.bool,
+};
+
+const defaultProps = {
+  children: null,
+  error: null,
+  help: null,
+  htmlFor: null,
+  isInline: false,
+  legend: null,
+  required: false,
+};
+
+const Fieldset = ({ children, error, help, htmlFor, isInline, legend, required, ...customProps }) => {
```

Comment:
It doesn't belong here. Removed.

https://github.com/cerner/terra-core/pull/352/commits/2c7f7c6030aea22430c5009637aca5681b94a9a6